packages/php-table-backend-utils/tests/Functional/Bigquery/Table/BigqueryTableReflectionTest.php

```diff
@@ -325,4 +379,200 @@ public function testIfTableDoesNotExists(): void
             $this->assertSame('Table "notExisting" not found.', $e->getMessage());
         }
     }
+
+    public function testTableWithPartitioning(): void
+    {
+        $dataset = $this->bqClient->createDataset(self::TEST_SCHEMA);
+        $dataset->createTable(
+            'test-partitions',
+            [
+                'schema' => [
+                    'fields' => [
+                        [
+                            'name' => 'id',
+                            'type' => 'BIGNUMERIC',
+                            'mode' => 'NULLABLE',
+                        ],
+                        [
+                            'name' => 'test',
+                            'type' => 'STRING',
+                            'mode' => 'NULLABLE',
+                            'maxLength' => '10',
+                        ],
+                    ],
+                ],
+                'timePartitioning' => [
+                    'type' => 'DAY',
+                    'expirationMs' => null,
+                    'field' => null,
+                ],
+                'rangePartitioning' => null,
+                'clustering' => [
+                    'fields' => [
+                        'test',
+                        'id',
+                    ],
+                ],
+                'requirePartitionFilter' => false,
+            ]
+        );
+
+        $rows = [];
+        foreach (range(1, 100) as $row) {
+            $rows[] = sprintf('(%d,\'test\')', $row);
+        }
+
+        $this->bqClient->runQuery($this->bqClient->query(sprintf(
+            <<<SQL
+INSERT %s.%s (`id`,`test`) VALUES %s;
+SQL,
+            BigqueryQuote::quoteSingleIdentifier(self::TEST_SCHEMA),
+            BigqueryQuote::quoteSingleIdentifier('test-partitions'),
+            implode(',', $rows)
+        )));
+        $ref = new BigqueryTableReflection($this->bqClient, self::TEST_SCHEMA, 'test-partitions');
+        $this->assertCount(1, $ref->getPartitionsList());
```

Comment:
nebylo by. tady fajn assertnout i obsah tech partitions z nasi reflection?

Reply:
no i kdyz to je pak testovane v testTableWithPartitioningConfig ...

Reply:
jop to sa testuje tam, v tom partition listu stejně není co assertovat možná počet rows, ale to mě přijde zbytečné, všecko si tam generuje dynamicky BQ.